hiba-chk.c

```diff
@@ -143,13 +147,13 @@ main(int argc, char **argv) {
 
 		if (host == NULL) {
 			host = hibacert_new();
-			if ((ret = hibacert_from_ext(host, identity, principal, 0, 42)) < 0)
+			if ((ret = hibacert_from_ext(host, identity, principal, 0, HOST_DEFAULT_CERT_SERIAL)) < 0)
```

Comment:
I don't quite remember why we're using a hard-coded serial here. Would it make sense to leave a comment explaining why this is fine?

Reply:
Done. Added a not in the man page too.